.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+.vscode
+coverage
+lib
```

Comment:
Should this really be ignored 🤔 ?

Reply:
Yes, lib is the place with transformed ts to js, so we shouldn't push it to git, only to npm.